src/Raven.Server/Documents/Patch/ScriptRunner.cs

```diff
@@ -316,7 +316,8 @@ public SingleRun(DocumentDatabase database, RavenConfiguration configuration, Sc
                     }
                     catch (Exception e)
                     {
-                        throw new JavaScriptParseException("Failed to parse: " + Environment.NewLine + script, e);
+                        if (key is not PatchRequest { Type: PatchRequestType.EtlBehaviorFunctions })
+                            throw new JavaScriptParseException("Failed to parse: " + Environment.NewLine + script, e);
```

Comment:
Can we pass `bool skipValidation` parameter instead and just omit `ScriptEngine.Execute(script);` which as I understand is called only for validation purposes?

Reply:
Okay, then maybe it would be better to have `bool ignoreValidationErrors` (optional parameter, default: false). Doing that based on the type of the script is a bit tricky and not visible on the caller side.